### ISSUE TITLE
[WIP] Enable users to add any credit card to bank account wallet.

### ISSUE THREAD

**Issue by vagnerwentz** [vagnerwentz/visualizesse#6]
When opening a bank account, we are presented with the option to acquire zero, one, or multiple credit cards linked to that account, facilitating various debit and credit transactions across these cards.

Using a credit card, you typically initiate purchases and settle payments on a date of your preference, such as the fifth business day of each month. Consequently, when recording a credit card transaction, the funds are not immediately subtracted from your account.

- [ ] Users should be able to establish a credit card linked to their bank account.
- [ ] Users should be able to designate a due date for bill payment.
- [ ] Transactions should include the credit card ID.
- [ ] Funds should not be deducted upon registering a credit card transaction.

**Comment by whoiswentz** [comment]
Remember security good practices here, never ask the user his entire credit card number, you can ask infos like nickname, four last digits, bank, and the "brand"

**Comment by vagnerwentz** [comment]
> Remember security good practices here, never ask the user his entire credit card number, you can ask infos like nickname, four last digits, bank, and the "brand"

Yes, the purpose of this feature for now is to allow the user to register a credit card with minimal information, solely to facilitate transactions made by credit card.
